lua/astrocommunity/completion/cmp-cmdline/init.lua

```diff
@@ -1,22 +1,28 @@
-local cmp = require "cmp"
-
 return {
   "hrsh7th/cmp-cmdline",
-  lazy = false,
-  opts = {
-    mapping = cmp.mapping.preset.cmdline(),
-    sources = cmp.config.sources({
-      { name = "path" },
-    }, {
-      {
-        name = "cmdline",
-        option = {
-          ignore_cmds = { "Man", "!" },
+  opts = function()
+    local cmp_mapping = require "cmp.config.mapping"
+    local cmp_sources = require "cmp.config.sources"
+
+    return {
+      mapping = cmp_mapping.preset.cmdline(),
+      sources = cmp_sources({
+        { name = "path" },
+      }, {
+        {
+          name = "cmdline",
+          option = {
+            ignore_cmds = { "Man", "!" },
+          },
         },
```

Comment:
Any particular reason why you changed the opts to a function?

Reply:
That's for further lazy loading the `cmp_sources` & `cmp_mappings`.